api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -200,6 +200,11 @@ model ApiProfile {
   /** The internet visibility of the OpenShift API server */
   @visibility(Lifecycle.Read, Lifecycle.Create)
   visibility?: Visibility = Visibility.Public;
+
+  /** The list of authorized IPv4 CIDR blocks allowed to access the API server. Maximum 5000 entries. */
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  @maxItems(5000)
```

Comment:
How does this match with the fact that if I understand correctly the corresponding attribute in Hypershift is https://github.com/openshift/hypershift/blob/2892457510cc7a86b5362ce10d0e8757a7e1fbec/api/hypershift/v1beta1/hostedcluster_types.go#L1082 which has a limit of 25 entries? is that going to be increased on that side too?

I don't think we are going to get an answer to this today so we can merge if needed but we should follow up on this as there seems to be a big disconnect